support-frontend/app/controllers/RedemptionController.scala

```diff
@@ -26,6 +28,10 @@ import views.html.subscriptionRedemptionForm
 
 import scala.concurrent.{ExecutionContext, Future}
 
+trait DynamoTableAsyncForUser {
```

Comment:
wasn't quite sure what to call this and couldn't find a precedent - I looked at the payment api service and it seems to just have it as a parameter on the actual method, Promotion service seems to use the TouchpointServiceProvider which handles test users and accessing the config together.
In the end I just called it what it is, get the DynamoTableAsync for user.